README.md

```diff
@@ -153,9 +153,15 @@ Valkyrie.config.metadata_adapter.query_service.find_all_of_model(model: MyModel)
 
 ## Installing a Development environment
 
-To run the test suite:
+### External Requirements
+* PostgreSQL with the uuid-ossp extension
+  * Databases named `Valkyrie_gem_development` and `Valkyrie_gem_test`.
```

Comment:
You can run `rake db:create` before `rake db:migrate` to have these created instead of making this a requirement.

Reply:
`db:create` did not work for me with default postgres privileges. I guess this can be a note that the db user needs to be able to do that?